sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -122,15 +129,18 @@ def __init__(
         # Although we might normally thread through **kwargs this causes
         # problems as MessageProperties won't absorb spurious args.
         self._encoding = kwargs.pop("encoding", "UTF-8")
-        self._uamqp_message = None
-
-        if "raw_amqp_message" in kwargs:
-            # Internal usage only for transforming AmqpAnnotatedMessage to outgoing ServiceBusMessage
-            self._raw_amqp_message = kwargs["raw_amqp_message"]
-        elif "message" in kwargs:
-            self._raw_amqp_message = AmqpAnnotatedMessage(message=kwargs["message"])
+        self._uamqp_message: Optional[Union[LegacyMessage, "Message"]] = None
+        self._message: Union["Message", "pyamqp_Message"] = None # type: ignore
```

Comment:
```suggestion
        self._message: Optional[Union["Message", "pyamqp_Message"]] = None # type: ignore
```

Reply:
I wasn't prefixing the uamqp.Message with `uamqp_` in only this file, b/c the `Message` type is exposed in the `message` property getter/setter, so wasn't sure whether we'd want to expose that `uamqp_Message = uamqp.Message` alias as well?

Reply:
prefixed all pyamqp types with `pyamqp_` as well